### ISSUE TITLE
I Don't Exist

### ISSUE THREAD

**Issue by VegetaDad** [al-caughey/Google-Meet-Attendance#109]
When I log in to a meet, I used to be the first one checked in. Now, periodically, I am not recognized and the app misses students. When this happens, all of the icons are stacked one on top of the other in a giant totem pole.
     I used to be able to clear this by popping out and back in again. Today, I went as far as closing Chrome and starting from scratch. It still isn't seeing me. As an intermittent issue, maybe I've just hit a bad streak.
     The next thing would be to reboot the entire machine but that isn't always a great option in the middle of the day.
     I have come to depend heavily on your app as it is by far the best that I have come across and functions in the most elegant manner. I'll let you know if I find a fix on this end but I'm thinking it may be beyond me.
Thanks,
Joe

**Comment by VegetaDad** [comment]
Today, I was not recognized until I opened the class list on the right side of the screen. Thank you for that fix.
When I closed the list, however, I got a ghost. I have been in the class the entire time but I have a ghost. (At least I'm there now.)
Something is still stacking everyone like a totem pole.

**Comment by VegetaDad** [comment]
[6th-f20 (2021-02-17).xlsx](https://github.com/al-caughey/Google-Meet-Attendance/files/5998178/6th-f20.2021-02-17.xlsx)
This is what my report looked like after class. I (Instructor) was there the entire time.

**Comment by Eagle-Man** [comment]
Were you previously using Grid View?  Grid View used to put yourself in the grid, but Meet's Tiled view does not.  The reason you're showing up with the people panel open is because you always appear as a participant there, but you're not registering with the People panel closed is because your video feed is not present.  If you look in the top right corner of Meet, you can turn on an option to add your tile, and then you should show up, even with the people panel closed.

**Comment by VegetaDad** [comment]
I stopped using Grid View and even removed it from Chrome. It didn't seem to be helping at all so I got id of it entirely.
I have my tile in the tiled view as per the button that appears over my picture on mouseover.
That does not help.
The attendance app still does not see me unless I open the show everyone/chat/activities panel. Oddly enough, I don't even have to open the show everyone bit to be seen. Chat or Activities are enough for the App to see me. Once I close that side panel, I get a ghost as though I left the meeting.

**Comment by VegetaDad** [comment]
Today:
Again, it would not see me unless I opened the attendance/chat/activities menu.
There was a new message in the roll list that said file an abuse report and had a ghost in it.
The app also did not save an html nor a csv file.

**Comment by VegetaDad** [comment]
2/23/21
It's getting worse.
I did get an HTML file today but that is because I selected "both" in the settings. I had previously just used the CSV and converted to Excel because that worked better for my method of bookkeeping.
The App has stopped saving CSV files altogether.
The HTML file is no longer alphabetizing correctly. The saved HTML list does not match the class list alphabetically.

![Meet Attendance Screenshot 02-23-21](https://user-images.githubusercontent.com/79174965/108871894-799ca380-75ae-11eb-9ac4-ec4d1ae2b108.jpg)

**Comment by AlCaughey** [comment]
Pls send me the file

**Comment by VegetaDad** [comment]
Not meaning to sound dim.

"the file"? Where do I find it and/or which one? The attendance report?

[image: Palmdale High School] <https://htmlsig.com/t/000001EHCJEJ>

Joseph "Joe" Pincetich | Director of Instrumental Music jpincetich@avhsd.org

Palmdale High School
(661) 273-3181 x803
2137 East Avenue R
Palmdale, CA 93550
https://www.palmdalehs.org/

[image: Twitter] <https://htmlsig.com/t/000001E926NX> [image: Facebook]
<https://htmlsig.com/t/000001E9M1BH> [image: Instagram]
<https://htmlsig.com/t/000001ECCHS6>

[image: htmlsig.com]

**Comment by AlCaughey** [comment]
Pls send the HTML report that is showing the errors... by default it gets
created in your downloads folder

**Comment by VegetaDad** [comment]
Here it is. It still has student names on it, sorry about that.


[image: Palmdale High School] <https://htmlsig.com/t/000001EHCJEJ>

Joseph "Joe" Pincetich | Director of Instrumental Music jpincetich@avhsd.org

Palmdale High School
(661) 273-3181 x803
2137 East Avenue R
Palmdale, CA 93550
https://www.palmdalehs.org/

[image: Twitter] <https://htmlsig.com/t/000001E926NX> [image: Facebook]
<https://htmlsig.com/t/000001E9M1BH> [image: Instagram]
<https://htmlsig.com/t/000001ECCHS6>

[image: htmlsig.com]

**Comment by VegetaDad** [comment]
The update helped.
App is now saving the CSV file.
Alphabetization is now correct.
The App still does not record me unless I open the Attendance/Chat/Activities window. I also disappear when that window is closed.
Just a note, when I open the Attendance window, two new icons appear (Add People and Host Controls). The Chat and activities windows are not changed.  I know that you are _not_ connected to the changes that Google makes to their meeting format. I just wanted to mention that change in case it effected your efforts.
Thank you again for a great product.

**Comment by Eagle-Man** [comment]
If you're using just the Tiled view, it's because the name on your video feed is "You" and not your name, whereas in the People list, it's your name.  This cannot be fixed as it's a Google thing.

**Comment by VegetaDad** [comment]
The app has stopped auto saving the CSV file.
It won't even save if I use the Save CSV button on top of the class list.

**Comment by VegetaDad** [comment]
App is not saving the CSV file even when the save CSV button is used.

**Comment by Eagle-Man** [comment]
I don't think anyone else has had this CSV issue.  Are you able to download the CSVs from the History?  If not, then it's likely an issue on your end with the way your computer/browser is set up.

For the "students," this is due to a change in Meet's code.  Check out Al's recent post about the latest version, as well as one specifically on the "and X more" entry to see how to exclude them.

For alphabetization, are you using a sort in the extension?  Do you have any specific entries for "last name" in the extension on any of your students?  Do these out-of-place students have anything in common in their names?  Otherwise, I think you would need to send the HTML file to Al to analyze.  Again, it doesn't seem to be an issue for anyone else, so it may be difficult to track down the cause of the issue.

**Comment by VegetaDad** [comment]
Again, not meaning to sound dim, where is the History from which I would download the CSV files (if they are there)?

I will check out the latest post on the "students" issue and see if I'm not already doing that.

I am going to look at the listing for the students and see if they all have the surname field filled in. That may solve the issue if there are problems.

Thanks

**Comment by Eagle-Man** [comment]
You didn't mention if you are using a sort on your class list.

For the history, at the bottom of the GMA pane there are three icons.  One for help (right), one for class list management (left), and once for "Past attendance info" according to the tooltip (middle).  If you click that middle icon, you will get a list of all the attendance sessions on record, and you will be able to select a session and redownload (recreate) the file using the typical save buttons.

**Comment by VegetaDad** [comment]
I found the history button. Thanks.

All but one of the missing CSV files that I have looked for were there.

The CSV did not download again this morning but it was in the history.
Thank you for showing me that.

I'm not sure what you mean by "using a sort". In the Settings area the
second setting is Sort Names By and I have the recommended button selected
for the last name. Should I set it to Manual?

GMA seems to be ignoring the Surname field and sorting by the display name
(At least it did that today).

[image: Palmdale High School] <https://htmlsig.com/t/000001EHCJEJ>

Joseph "Joe" Pincetich | Director of Instrumental Music jpincetich@avhsd.org

Palmdale High School
(661) 273-3181 x803
2137 East Avenue R
Palmdale, CA 93550
https://www.palmdalehs.org/

[image: Twitter] <https://htmlsig.com/t/000001E926NX> [image: Facebook]
<https://htmlsig.com/t/000001E9M1BH> [image: Instagram]
<https://htmlsig.com/t/000001ECCHS6>

[image: htmlsig.com]

**Comment by Eagle-Man** [comment]
> Should I set it to Manual?
No, that should work.  It SHOULD sort by whatever's after the last space in their display name, unless something is present in the Surname field for the student in GMA, in which case it should use that for sorting.

Also, Al just posted something on the Facebook page about fixing the no CSV issue.
